boot/freeldr/freeldr/arch/realmode/helpers_all.inc

```diff
@@ -0,0 +1,70 @@
+/*
+ * PROJECT:     FreeLoader
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Real mode helper code for All PCs
```

Comment:
"all PCs" ? do you mean "all architectures" (PC-AT, NEC PC98, etc.)?

Reply:
yes!